src/NUnitFramework/framework/Interfaces/TNode.cs

```diff
@@ -22,54 +22,44 @@
 // ***********************************************************************
 
 using System;
+using System.Xml;
+
+#if PORTABLE || SILVERLIGHT
+using System.Xml.Linq;
+#endif
 
 namespace NUnit.Framework.Interfaces
 {
     /// <summary>
-    /// XmlNode represents a single node in the XML representation
+    /// TNode represents a single node in the XML representation
     /// of a Test or TestResult. It replaces System.Xml.XmlNode and
-    /// provides a minimal set of methods for operating on the XML 
-    /// in a platform-independent manner.
+    /// System.Xml.Linq.XElement, providing a minimal set of methods 
+    /// for operating on the XML in a platform-independent manner.
     /// </summary>
-    public class XmlNode
+    public class TNode
```

Comment:
I modelled it after XNode and thought of it as a T(est)Node. I'm not 100% happy with the name but I can't think of anything better that won't conflict with the System.Xml or System.Xml.Linq namespeaces. Suggestions?